fe/fe-core/src/main/java/com/starrocks/connector/parser/trino/AstBuilder.java

```diff
@@ -999,6 +999,11 @@ protected ParseNode visitCoalesceExpression(CoalesceExpression node, ParseTreeCo
     @Override
     protected ParseNode visitExtract(Extract node, ParseTreeContext context) {
         String fieldString = node.getField().toString();
+        if (fieldString.equalsIgnoreCase("dow")) {
```

Comment:
you can use `Trino2SRFunctionCallTransformer.convert(node.getName().toString(), arguments)`,  no need transform function here

Reply:
I don't quite understand, can you describe it in detail @Youngwb

Reply:
```
protected ParseNode visitExtract(Extract node, ParseTreeContext context) {
        String fieldString = node.getField().toString().toLowerCase();
        return  Trino2SRFunctionCallTransformer.convert(fieldString,
                Lists.newArrayList((Expr) visit(node.getExpression(), context)));
    }
```

Reply:
ths